uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/entitlement-instance_test.js

```diff
@@ -0,0 +1,44 @@
+define([
+  'leave-absences/shared/apis/entitlement-api',
+  'leave-absences/shared/models/entitlement-model',
+  'leave-absences/shared/models/instances/entitlement-instance',
+], function () {
+  'use strict'
+
+  describe('EntitlementInstance', function () {
+    var Entitlement, EntitlementInstance, $rootScope, $httpBackend, EntitlementAPIMock;
+
+    beforeEach(module('leave-absences.models'));
+
```

Comment:
after var declaration there is a new line and after beforeEach ie., inbetween 2 beforeEach there is white line